modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonFastAPIServerCodegen.java

```diff
@@ -148,6 +149,10 @@ public void processOpts() {
             setPackageName((String) additionalProperties.get(CodegenConstants.PACKAGE_NAME));
         }
 
+        if (additionalProperties.containsKey(CodegenConstants.PACKAGE_VERSION)) {
+            setPackageVersion((String) additionalProperties.get(CodegenConstants.PACKAGE_VERSION));
+        }
```

Comment:
what about adding a `else` to set the default of `packageVersion` to `appVersion` for backward compatibility?

Reply:
appVersion is obtained from (or default to) the value in the spec: https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/test/resources/3_0/petstore.yaml#L8